docs/markdown/Build-options.md

```diff
@@ -16,18 +16,20 @@ Here is a simple option file.
 option('someoption', type : 'string', value : 'optval', description : 'An option')
 option('other_one', type : 'boolean', value : false)
 option('combo_opt', type : 'combo', choices : ['one', 'two', 'three'], value : 'three')
+option('array_opt', type : 'stringarray', value : ['one', 'two'])
```

Comment:
`s/value/choices/`, right?

Reply:
Oh hold on, looking at the implementation, you meant this to be a an array of free strings, not validated, hence the difference with #2390
I'm not sure what the use for this is, but sure, that looks alright then :)